starknet_py/net/models/chains.py

```diff
@@ -10,10 +13,35 @@ class StarknetChainId(Enum):
     TESTNET = int_from_bytes(b"SN_GOERLI")
     TESTNET2 = int_from_bytes(b"SN_GOERLI2")
 
+    @classmethod
+    def from_network(cls, net: Network) -> StarknetChainId:
+        """
+        Create a chain from given network.
+        :raises ValueError: when Network is unknown.
+        :param net: Network of the chain.
+        :return: StarknetChainId instance.
+        """
+        net_to_chain = {
+            MAINNET: cls.MAINNET,
+            TESTNET: cls.TESTNET,
+            TESTNET2: cls.TESTNET2,
+        }
+
+        if not isinstance(net, str) or net not in net_to_chain:
+            raise ValueError(
+                f"Cannot create StarknetChainId from unknown Network '{net}'."
```

Comment:
```suggestion
                f"Cannot create StarknetChainId from unknown net '{net}'."
```

I think it's more descriptive if we use an argument name instead of a type.